README.md

```diff
@@ -1,3 +1,27 @@
-# Plain HTML Pages Template
+# SSE-Notes
 
-This is a template that uses plain HTML to create a static web page that is published at git-pages.thm.de.
+This is a project for our Secure Software Engineering course.
+
+It consists of a server and a client which make the writing and saving of private and public markdown notes possible.
+
+## Current State Of The Project
+[![CodeQL](https://github.com/BranAndSceolan/SSE-Notes/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/BranAndSceolan/SSE-Notes/actions/workflows/codeql-analysis.yml)
+
+## Building This Project
+
+### Backend
+* go into the backend directory ``cd backend``
+* make sure all dependencies are installed ``npm install``
+* either:
+  * build for development: 
+    * create a .env file with
+      * NOTES_PASSWORD=choose_a_password
+      * POSTGRES_USER=choose_an_admin_name
```

Comment:
Why does the backend need/use the admin account?

Reply:
The init-script used in the Dockerfile uses the admin account, so I thought backend needs it to construct the database container.